azure_iot_edge/README.md

```diff
@@ -0,0 +1,146 @@
+# Agent Check: Azure IoT Edge
+
+## Overview
+
+[Azure IoT Edge][1] is a fully managed service to deploy Cloud workloads to run on Internet of Things (IoT) edge devices via standard containers.
+
+Use the Datadog-Azure IoT Edge integration to collect metrics and health status from IoT Edge devices.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an IoT Edge device running on a device host.
+
+### Installation
+
+The Azure IoT Edge check is included in the [Datadog Agent][2] package.
+
+No additional installation is needed on your device.
+
+### Configuration
+
+It is recommended to configure the IoT Edge device so that the Agent runs as a custom module. Follow the official Microsoft documentation on [deploying Azure IoT Edge modules][3] for general information on installing and working with custom modules for Azure IoT Edge.
+
+Follow the steps below to configure the IoT Edge device, runtime modules, and the Datadog Agent to start collecting IoT Edge metrics.
+
+1. Edit your IoT Edge device `config.yaml` file:
+    - **Linux**: Make sure the `connect.management_uri` and `listen.management_uri` options point to a Unix Domain Socket (note that this is the default configuration). For example:
+
+        ```yaml
+        # /etc/iotedge/config.yaml
+
+        connect:
+            management_uri: "unix:///var/run/iotedge/mgmt.sock"
+            # ...
+
+        listen:
+            management_uri: "fd://iotedge.mgmt.socket"
```

Comment:
Yes. The default is `fd://...` which actually corresponds to the `systemd` format. (The Linux Security Manager is actually just a systemd service, and it does its own socket binding thing, this is all related to how the Security Manager is configured by default. I can dive into specifics if needed).

I preferred to keep the default config rather than requiring users to set eg `/var/run/iotedge/mgmt.sock` explicitly, although I guess that's a possible option (but that would mean an extra thing for users to tweak). The config listed in the Linux case here is 100% the default one, untouched, which is why I wrote "Make sure that…".

Reply:
Now irrelevant since this config was dropped, as we dropped the security manager service check.